charts/gatekeeper/templates/gatekeeper-validating-webhook-configuration-validatingwebhookconfiguration.yaml

```diff
@@ -31,10 +31,13 @@ webhooks:
     operations:
     - CREATE
     - UPDATE
+    {{- if .Values.enableDeleteOperations }}
```

Comment:
same here, you can revert any changes in `charts`

Reply:
Even though I don't fully understand (isn't there a discrepancy now between the chart in `charts` and in `manifest_staging`?), I have reverted all changes in `charts`.

Reply:
when we cut a new release, everything existing in `charts` is wiped and `manifest_staging/charts` gets promoted to `charts`.